src/eos/eosEngine.ts

```diff
@@ -198,15 +198,6 @@ export class EosEngine extends CurrencyEngine<EosTools, SafeEosWalletInfo> {
     opts: EdgeCurrencyEngineOptions
   ): Promise<void> {
     await super.loadEngine(plugin, walletInfo, opts)
-
-    if (typeof this.walletInfo.keys.ownerPublicKey !== 'string') {
-      if (walletInfo.keys.ownerPublicKey != null) {
-        this.walletInfo.keys.ownerPublicKey = walletInfo.keys.ownerPublicKey
-      } else {
-        const pubKeys = await plugin.derivePublicKey(this.walletInfo)
-        this.walletInfo.keys.ownerPublicKey = pubKeys.ownerPublicKey
-      }
-    }
```

Comment:
Ahhh, this was before the days where the core did the public key derivation for us. Yes, remove it.

Reply:
This code predates `derivePublicKey`?

Reply:
I believe so, yes. Early on, engines would just get the private keys, and were on their own in terms of deriving public keys. This was slow, and everybody did it differently, so we moved it to the core. A few old coins still do it themselves, redundantly.